lib/cylc/cfgspec/globalcfg.py

```diff
@@ -127,6 +127,17 @@
             'retrieve job logs max size'  : vdr( vtype='string' ),
             'retrieve job logs retry delays': vdr( vtype='interval_minutes_list', default=[] ),
             'task event handler retry delays': vdr( vtype='interval_minutes_list', default=[] ),
+            'local tail command template' : vdr( vtype='string', default="tail -n +1 -F %(filename)s"),
+            'remote tail command template' : vdr( vtype='string', default="tail --pid=`ps h -o ppid $$ | sed -e s/[[:space:]]//g` -n +1 -F %(filename)s"),
+            # Template for tail commands on remote files.
```

Comment:
Now that you mention it, I have just tested the command on the various platforms. It seems to work on SuSE (which is the our main platform requiring this functionality) but not on RedHat!